### PR TITLE
Handle unhandled errors

### PR DESCRIPTION
**Pull request by NathanBnm**
There were many warnings about unhandled `GLib.IOError` and `GLib.DBusError`. This PR resolves this.

### CODE REVIEW

File: src/Services/DBusManager.vala [hunk 1]
```diff
@@ -47,6 +47,8 @@ namespace Hourglass.Services {
 
             } catch (IOError e) {
```
Comment (ryonakano) [inline src/Services/DBusManager.vala:48]:
```suggestion
            } catch (GLib.IOError e) {
```

While you're here :pray:

File: src/Services/DBusManager.vala [hunk 2]
```diff
@@ -47,6 +47,8 @@ namespace Hourglass.Services {
...
                 error ("%s", e.message);
+            } catch (GLib.DBusError e) {
+                error ("%s", e.message);
```
Comment (ryonakano) [inline src/Services/DBusManager.vala:51]:
```suggestion
                error (e.message);
```

`e.message` is string so we don't need to explicit the format specifier.

Comment (NathanBnm) [reply src/Services/DBusManager.vala:51]: Thanks for noticing

**Comment by ryonakano** [comment]
#77 was merged so this's ready to go :rocket:

**Pull request merged**